atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1119,10 +1119,11 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
         return estimatedSize;
     }
 
-    private <T> void getWithPostFiltering(TableReference tableRef,
-                                          Map<Cell, Value> rawResults,
-                                          @Output ImmutableMap.Builder<Cell, T> results,
-                                          Function<Value, T> transformer) {
+    private <T, R extends ImmutableMap.Builder<Cell, T>> R getWithPostFiltering(
+            TableReference tableRef,
+            Map<Cell, Value> rawResults,
+            R results,
```

Comment:
Hmm, `@Output` isn't exactly right but I'd still suggest having some form of documentation here (possibly in variable names or otherwise) that this will be modified and used to return the output

Reply:
Renamed to `resultsAccumulator`.